packages/zoe/src/instanceRecordStorage.js

```diff
@@ -0,0 +1,102 @@
+// @ts-check
+
+import { assert, details as X, q } from '@agoric/assert';
+import { assertKeywordName, getKeywords } from './cleanProposal';
+
+/**
+ * @callback AddIssuerToInstanceRecord
+ *
+ * Add an issuer and its keyword to the instanceRecord for the
+ * contract instance
+ *
+ * @param {Keyword} keyword
+ * @param {IssuerRecord} issuerRecord
+ * @returns {void}
+ */
+
+/**
+ * @callback GetInstanceRecord
+ * @returns {InstanceRecord}
+ */
+
+/**
+ *
+ * @typedef {{ addIssuerToInstanceRecord: AddIssuerToInstanceRecord,
+ * getInstanceRecord: GetInstanceRecord, getTerms: () => Terms,
+ * getIssuers: () => IssuerKeywordRecord,
+ * getBrands: () => BrandKeywordRecord, assertUniqueKeyword: (keyword:
+ * Keyword) => void }} InstanceRecordManager
```

Comment:
```suggestion
 * @typedef {{ addIssuerToInstanceRecord: AddIssuerToInstanceRecord,
 * getInstanceRecord: GetInstanceRecord, 
 * getTerms: () => Terms,
 * getIssuers: () => IssuerKeywordRecord,
 * getBrands: () => BrandKeywordRecord, 
 * assertUniqueKeyword: (keyword: Keyword) => void }} InstanceRecordManager
```

Since it is in a comment, prettier won't mess it up.